e2e/integrations/integration.steps.ts

```diff
@@ -107,8 +108,18 @@ class IntegrationSteps {
     const page = stepFactory.getStep(stepType, paremeter);
     return page.fillConfiguration();
   }
+
+  // Twitter search specification
+  @then (/^she fills "([^"]*)" details$/)
+  public fillKeywords(keywords: string): P<any> {
+
+    const keyWordsInputSelector = "#".concat(keywords);
+    const keyWordsInput = element(by.css(keyWordsInputSelector));
```

Comment:
Is there any better way to select to input field? If not and we're missing a meaningful ID on a div(s), let  me know and I will follow up with Eng team.